### ISSUE TITLE
OpenAPI, RAML, Avro, GraphQL, Protobuf, and XSD users reuse their schema/type definitions within AsyncAPI.

### ISSUE THREAD

**Issue by fmvilas** [asyncapi/shape-up-process#56]

**Comment by siathalysedI** [comment]
We have the same issue at our work so internally we came up with a custom solution to merge all of the schema registry into a single json and attach this into a custom validator to ease the validation process and at the same time make it faster. So I created a discussion thread

https://github.com/asyncapi/community/discussions/97

Any comments are welcome

**Comment by seungjinRonin** [comment]
Does this issue include the goal of exporting the *.yml contract as a `.proto` file?

**Comment by fmvilas** [comment]
No, the idea of this issue is to let users reference `.proto` (and other) files inside an AsyncAPI document. And tools should understand it.
